rpa-builder/6.1/modules/ROOT/pages/toolbox-mail-operations-mail-session-outlook-with-oauth.adoc

```diff
@@ -0,0 +1,196 @@
+= Mail Session (Outlook with OAuth)
+
+The _Mail Session (Outlook with OAuth)_ enables you to establish a connection to a mail server using MS Outlook and then use mail operations as reading mail (*Read Mail*), sending mail (*Send Mail*), and changing e-mails (*Set Mail*), by using the corresponding Action Steps
+
+== Before you Begin
+
+You need to have an Azure account to create an application for Outlook.
+
+[[properties]]
+== Properties
+
+* *OAuth Authentication* Settings
+** *OAuth Host*
++
+The host address that is used to get the credentials. The default address for Outlook services is https://login.microsoftonline.com/common/oauth2/v2.0/ . This address might change if the customer uses a self hosted service.
+** *Client ID*
++
+In order to authenticate with the OAuth Host, an Azure application has to be defined in the customers space. This Azure application has an id, that is unique in the entire OAuth Host space and allows a user to explicitly grant or revoke access for this Azure application to his account. For more information how to create this Azure application for Outlook, see here: https:// learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app
+** *Client Secret*
++
+This secret allows Design Studio to prove to the OAuth Host, that it has received the permission to access user accounts on behalf of the registered Azure application. Without it the authentication is not possible.
+** *Redirect Uri*
```

Comment:
URI should be written in uppercase letters: https://learn.microsoft.com/en-us/azure/active-directory/develop/reply-url